### DISCUSSION TITLE
Change icon styles

### DISCUSSION THREAD

**Discussion by ryanmitchell** [laravel/nova-issues#3620] [Q&A] [answered]
Is there a way of changing the icon SVGs for the edit / delete buttons and other icons in the system? I see they are available as Vue components, but if I change them then I lose the ability to update the core codebase.

Can they be overwritten or extended somehow?

**Comment by davidhemphill** [comment, marked as answer]
You can register a component to replace it with `Vue.component('icon-name', require('IconName').default)`

**Reply by ryanmitchell** [reply]
Thanks!
